RegisterPrestaShopDeveloperConference2024.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: Register Now for the PrestaShop Developer Conference 2024
+subtitle: CONFERENCE REGISTRATION IS NOW OPEN
+date: 2024-10-01 09:55:00
+authors: [ PrestaShop ]
+icon: icon-leaf
+tags:
+- news
+- event
+- community
+---
+
+The world of ecommerce is constantly evolving, and staying on top of the latest trends and innovations is essential for developers and businesses. That’s why we are thrilled to [invite you](https://events.prestashop.com/prestashop-developer-conference/fr) to the second edition of the PrestaShop Developer Conference, taking place on November 6, 2024, in Paris. 
+
+![Register_Developer_Conference](/assets/images/2024/10/RegisterDeveloperConference24.jpeg)
```

Comment:
I think this image would be better: https://pbs.twimg.com/media/GYywfD5XAAMDSFe?format=jpg&name=medium